src/main/java/seedu/taassist/storage/JsonAdaptedStudent.java

```diff
@@ -28,21 +28,21 @@ class JsonAdaptedStudent {
     private final String phone;
     private final String email;
     private final String address;
-    private final List<JsonAdaptedModuleClass> classes = new ArrayList<>();
+    private final List<JsonAdaptedModuleClass> moduleClasses = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedStudent} with the given student details.
      */
     @JsonCreator
     public JsonAdaptedStudent(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedModuleClass> classes) {
+            @JsonProperty("tagged") List<JsonAdaptedModuleClass> moduleClasses) {
```

Comment:
```suggestion
            @JsonProperty("classes") List<JsonAdaptedModuleClass> moduleClasses) {
```

Would something like this not work? Just wondering.

Reply:
I think some of the test cases use the "tagged" property instead of "classes" property. If we make the change here, we will have to change all the dummy code as well. Hope we can address this in issue #68 instead. :)